src/extension.ts

```diff
@@ -45,6 +46,16 @@ const draft = draftCreate(host, fs, shell);
 const configureFromClusterUI = configureFromCluster.uiProvider(fs, shell);
 const createClusterUI = createCluster.uiProvider(fs, shell);
 
+const deleteMessageItems: vscode.MessageItem[] = [
+    {
+        title: "Delete"
+    },
+    {
+        title: "Close",
+        isCloseAffordance: true
```

Comment:
Hah, I didn't know you could override the Close button this way!  Nice find!  I suggest you name the button Cancel instead of Close in that case?  There are a couple of other places I could do with doing something similar - thanks for finding it!